packages/react-components/src/echarts/themes/registerCloudscapeThemes.ts

```diff
@@ -255,7 +255,7 @@ export const registerCloudscapeThemes = () => {
       axisLine: {
         show: true,
         lineStyle: {
-          color: '#e0e2e7',
+          color: '#404d5c',
```

Comment:
Shouldn't we use the [tokens](https://cloudscape.design/foundation/visual-foundation/design-tokens/#tokens) so that these don't break with cloudscape updates?

Reply:
sadly we cant directly use the tokens because they are CSS variables and echarts will only accept the actual color as hex or name :( this is a big issue that we've run into when integrating echarts w cloudscape